test.py

```
@@ -0,0 +1,17 @@
+def greet(name):
+    print(f"Hello, {name}!")
+
+def calculate_sum(a, b):
```

Ruff

Missing return type annotation for public function `calculate_sum`

(ANN201)